data/data/aws/vpc/master-elb.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_lb" "api_internal" {
-  name                             = "${var.cluster_name}-int"
+  name                             = "${var.unique_cluster_name}-int"
```

Comment:
So looks like [name](https://www.terraform.io/docs/providers/aws/r/elb.html#name) is optional.
What if we drop the name here and add `Name` tag to the elb. And [tag](https://docs.aws.amazon.com/elasticloadbalancing/2012-06-01/APIReference/API_Tag.html) values can be max length 256?
is it possible?

Reply:
Hmm looks like that might be because of https://github.com/openshift/cluster-api-provider-aws/blob/57f3ee8bd749cc7522252302b38624fc8d762891/pkg/apis/awsproviderconfig/v1beta1/awsmachineproviderconfig_types.go#L270
:(